test/api/v3/integration/chat/POST-chat.test.js

```diff
@@ -69,6 +70,29 @@ describe('POST /chat', () => {
     });
   });
 
+  it('returns an error when chat message contains a banned word in tavern', async () => {
+    await expect(user.post(`/groups/habitrpg/chat`, { message: testBannedWordMessage})).to.eventually.be.rejected.and.eql({
+      code: 400,
+      error: 'BadRequest',
+      message: t('bannedWordUsed'),
+    });
+  });
+
+  it('does not error when sending a chat message containing a banned word to a private group', async () => {
+    let { group, members } = await createAndPopulateGroup({
+      groupDetails: {
+        name: 'Party',
+        type: 'party',
+        privacy: 'private',
+      },
+      members: 1,
+    });
+
+    let message = await members[0].post(`/groups/${group._id}/chat`, { message: testBannedWordMessage});
+
+    expect(message.message.id).to.exist;
+  });
+
```

Comment:
Can there also be tests for these cases?
- Does not error when posting to public guild
- Does not error when posting to private guild
- Does not error when the banned word is part of another word, e.g., `{ message: 'prefix' + testBannedWordMessage}` and `{ message: testBannedWordMessage + 'suffix'}`  (no space after prefix or before suffix)
- Does error when the banned word is part of a phrase, e.g., `{ message: 'first words ' + testBannedWordMessage + ' last words'}` (spaces around the banned word)